frontend/src/components/Dashboard.js

```diff
@@ -39,11 +46,29 @@ class Dashboard extends React.Component {
 
   async componentDidMount() {
     const res = await getAllDocuments()
+    const res2 = await getAllMessages()
+    const res3 = await getAllInformation()
```

Comment:
I would make these variables more descriptive, or add comments later on that describes the logic with these variables.